nltk/classify/rte_classify.py

```diff
@@ -32,7 +32,7 @@ def ne(token):
         return True
     return False
 
-def lemmatize(word):
+def _lemmatize(word):
```

Comment:
Out of curiosity I also wrote a query looking for all instantiation of the `RTEFeatureExtractor` class and again it appears it only occurs in `rte_classify.py` so modifying the __init__ method would also be fine.